packages/templates/wasm/assemblyscript/src/__tests__/e2e/integration.spec.ts

```diff
@@ -1,102 +1,30 @@
 import { PolywrapClient } from "@polywrap/client-js";
-import {
-  buildWrapper,
-  initTestEnvironment,
-  stopTestEnvironment,
-  providers,
-  ensAddresses
-} from "@polywrap/test-env-js";
 import * as App from "../types/wrap";
 import path from "path";
 
-import { getPlugins } from "../utils";
+jest.setTimeout(60000);
 
-jest.setTimeout(500000);
+describe("JSON RPC Wasm Wrapper (Rust)", () => {
```

Comment:
Why are we calling it JSON RPC Wasm Wrapper?

Reply:
i changed it